package.json

```
@@ -99,7 +100,7 @@
     "tailwindcss-vanilla-rtl": "^0.2.0",
     "ts-jest": "^29.1.0",
     "ts-loader": "^9.2.6",
```

Looks like we need to update `ts-loader` as well to get the build working again:

```suggestion
    "ts-loader": "^9.5.1",
```